jazzmin/static/jazzmin/js/ui-builder.js

```diff
@@ -140,9 +140,23 @@
                 )
             );
         });
+
+        $("#jazzmin-theme-chooser").on('change', function () {
+            const $themeCSS = $('#jazzmin-theme');
+            const currentSrc = $themeCSS.attr('href');
+            const currentTheme = currentSrc.split('/')[4];
+            const newTheme = $(this).val();
+            $themeCSS.attr('href', currentSrc.replace(currentTheme, newTheme));
+
+            if (newTheme === "darkly") {
```

Comment:
If certain UI tweaks are recommended with a theme, I'm selecting them here, e.g a dark navbar

Reply:
wouldn't this be recommended to all dark themes then?